content/installation/Discord/_index.md

```diff
@@ -87,21 +99,24 @@ Follow the first 4 mins of this [Video Tutorial](https://youtu.be/8o25pRbXdFw) t
 - Add **botkube** chart repository:
 
   ```bash
-  $ helm repo add botkube https://charts.botkube.io
-  $ helm repo update
+  helm repo add botkube https://charts.botkube.io
+  helm repo update
   ```
 
 - Deploy BotKube backend using **helm install** in your cluster:
 
   ```bash
-  $ helm install --version v0.12.4 botkube --namespace botkube --create-namespace \
-  --set communications.discord.enabled=true \
-  --set communications.discord.channel=<DISCORD_CHANNEL_ID> \
-  --set communications.discord.botid=<DISCORD_BOT_ID> \
-  --set communications.discord.token=<DISCORD_TOKEN> \
-  --set config.settings.clustername=<CLUSTER_NAME> \
-  --set config.settings.kubectl.enabled=<ALLOW_KUBECTL> \
-  --set image.tag=v0.12.4 \
+  export CLUSTER_NAME={cluster_name}
+  export ALLOW_KUBECTL={allow_kubectl}
+
+  helm install --version v0.13.0 botkube --namespace botkube --create-namespace \
+  --set communications.default-group.discord.enabled=true \
+  --set communications.default-group.discord.channels.default.id=${DISCORD_CHANNEL_ID} \
+  --set communications.default-group.discord.botID=${DISCORD_BOT_ID} \
+  --set communications.default-group.discord.token=${DISCORD_TOKEN} \
+  --set settings.clusterName=${CLUSTER_NAME} \
+  --set executors.kubectl-read-only.kubectl.enabled=${ALLOW_KUBECTL} \
+  --set image.tag=v0.13.0 \
```

Comment:
Question: Do we want to define override for image tag, if the default value inside `v0.13.0` Helm chart will have proper one already set? I think this is not necessary 🤔

Reply:
good point 👍 totally agree :)